digid_eherkenning/views/base.py

```diff
@@ -18,3 +20,9 @@ def get_redirect_url(request, redirect_to, require_https=True):
         return redirect_to
 
     return ""
+
+
+def is_relative_path(uri: str) -> bool:
+    """Check if `uri` is a relative path."""
+    parsed = urlparse(uri)
+    return not (parsed.scheme or parsed.netloc)
```

Comment:
here be dragons - please check the comments in the django source code for `_url_has_allowed_host_and_scheme`!

Reply:
> f the mock IDP is currently hosted under regular HTTP

hosting anything not under HTTPS and throwing pentests at it is an exercise in futility anyway, so acknowledged but dismissed :-)